frameworks/helloworld/integration/tests/test_sanity.py

```diff
@@ -71,6 +72,37 @@ def test_bump_hello_nodes():
     tasks_not_updated('hello', hello_ids)
 
 
+@pytest.mark.sanity
+def test_lock():
+    '''This test verifies that a second scheduler fails to startup when
+    an existing scheduler is running.  Without locking, the scheduler
+    would fail during registration, but after writing its config to ZK.
+    So in order to verify that the scheduler fails immediately, we ensure
+    that the ZK config state is unmodified.'''
+
+    zk_path = "dcos-service-{}/ConfigTarget".format(PACKAGE_NAME)
+
+    # Get ZK state from running framework
+    zk_config_old = shakedown.get_zk_node_data(zk_path)
+
+    # Install second framework
+    options = {"service": {"name": "hello-world-lock"}, "hello": {"count": 2}}
```

Comment:
"hello-world-lock" is just the service name for this test, like "kafka".  So whatever dcos-commons does normally to name the ZK directory is what it does here.  The lock itself is a child znode.